packages/webdriverio/src/utils/implicitWait.js

```diff
@@ -0,0 +1,30 @@
+import logger from '@wdio/logger'
+const log = logger('webdriverio')
+
+export default async function implicitWait (currentElement, commandName) {
+    /**
+     * wait on element if:
+     *  - elementId couldn't be fetched in the first place
+     *  - command is not explicit wait command for existance or displayedness
+     */
+
```

Comment:
```suggestion
```

Reply:
Also the comment above can be moved on top of the function declaration

Reply:
Fixed.